py/desitarget/commissioning/commissioning_cuts.py

```diff
@@ -0,0 +1,2524 @@
+
+import astropy.units as u
+from astropy.coordinates import SkyCoord
+from astropy.table import Table, Row
+
+from desitarget import io
+from desitarget.internal import sharedmem
+import desitarget.targets
+from desitarget.targetmask import desi_mask, bgs_mask, mws_mask
+
+from desitarget.gaiamatch import match_gaia_to_primary, pop_gaia_coords
+
+#ADM set up the DESI default logger
+from desiutil.log import get_logger
+log = get_logger()
+
+#ADM start the clock
+start = time()
+
+def _gal_coords(ra,dec):
+    """Shift RA, Dec to Galactic coordinates
+
+    Parameters
+    ----------
+    ra, dec : :class:`array_like` or `float`
+        RA, Dec coordinates (degrees)
+
+    Returns
+    -------
+    The Galactic longitude and latitude (l, b)
+
+    """
+
+    c = SkyCoord(ra*u.deg, dec*u.deg)
+    gc = c.transform_to('galactic')
+
+    return gc.l.value, gc.b.value    
+
+
+def shift_photo_north_pure(gflux=None, rflux=None, zflux=None):
+    """Same as :func:`~desitarget.cuts.shift_photo_north_pure` accounting for zero fluxes
+
+    Parameters
+    ----------
+    gflux, rflux, zflux : :class:`array_like` or `float`
+        The flux in nano-maggies of g, r, z bands.
+
+    Returns
+    -------
+    The equivalent fluxes shifted to the southern system.
+
+    Notes
+    -----
+        - see also https://desi.lbl.gov/DocDB/cgi-bin/private/RetrieveFile?docid=3390;filename=Raichoor_DESI_05Dec2017.pdf;version=1
+
+    """
+
+    gshift = gflux * 10**(-0.4*0.029) * (gflux/rflux)**(-0.068)
+    rshift = rflux * 10**(+0.4*0.012) * (rflux/zflux)**(-0.029)
+    zshift = zflux * 10**(-0.4*0.000) * (rflux/zflux)**(+0.009)
+
+    return gshift, rshift, zshift
+
+
+def shift_photo_north(gflux=None, rflux=None, zflux=None):
+    """Convert fluxes in the northern (BASS/MzLS) to the southern (DECaLS) system
+
+    Parameters
+    ----------
+    gflux, rflux, zflux : :class:`array_like` or `float`
+        The flux in nano-maggies of g, r, z bands.
+
+    Returns
+    -------
+    The equivalent fluxes shifted to the southern system.
+
+    Notes
+    -----
+        - see also https://desi.lbl.gov/DocDB/cgi-bin/private/RetrieveFile?docid=3390;filename=Raichoor_DESI_05Dec2017.pdf;version=1
+
+    """
+    
+    #ADM only use the g-band color shift when r and g are non-zero
+    gshift = gflux * 10**(-0.4*0.029)
+    w = np.where( (gflux != 0) & (rflux != 0) )
+    gshift[w] = (gflux[w] * 10**(-0.4*0.029) * (gflux[w]/rflux[w])**complex(-0.068)).real
+
+    #ADM only use the r-band color shift when r and z are non-zero
+    #ADM and only use the z-band color shift when r and z are non-zero
+    w = np.where( (rflux != 0) & (zflux != 0) )
+    rshift = rflux * 10**(+0.4*0.012)
+    zshift = zflux * 10**(-0.4*0.000)
+
+    rshift[w] = (rflux[w] * 10**(+0.4*0.012) * (rflux[w]/zflux[w])**complex(-0.029)).real
+    zshift[w] = (zflux[w] * 10**(-0.4*0.000) * (rflux[w]/zflux[w])**complex(+0.009)).real
+
+    return gshift, rshift, zshift
+
```

Comment:
When we actually have the cuts, we should import utility functions from `desitarget.cuts` (such as the "shift" function you note) but we should *not* import any of the actual functions that perform the cuts, in case we later change code in `desitarget.cuts` for the main survey.